### PR TITLE
docs: adds OR sample

### PR DESCRIPTION
**Pull request by telpirion**

### CODE REVIEW

File: samples/queryFilterOr.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+async function main() {
+  // [START datastore_query_filter_or]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = "your Google Cloud project id";
+
+  // Imports the Cloud Datastore
+  const {Datastore, PropertyFilter, or} = require('@google-cloud/datastore');
```
Comment (danieljbruce) [inline samples/queryFilterOr.js:29]: This is great. Now these functions are directly available to the user.

File: samples/queryFilterOr.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  async function queryFilterOr() {
+    // Instantiate the Datastore
+    const datastore = new Datastore();
+    const query = datastore
+      .createQuery('Task')
+      .filter(
+        or([
```
Comment (danieljbruce) [inline samples/queryFilterOr.js:37]: Look good!

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (danieljbruce) [inline .vscode/launch.json:1]: Should we remove this file and the protos?

**Review by danieljbruce — APPROVED** [review]

**Comment by telpirion** [comment]
Thank you kindly!

**Pull request merged**